contentcuration/contentcuration/static/js/edit_channel/models.js

```diff
@@ -550,6 +565,46 @@ var ChannelModel = BaseModel.extend({
             });
         });
     },
+    get_node_diff: function(){
+        var self = this;
+        return new Promise(function(resolve, reject){
+            $.ajax({
+                method:"POST",
+                url: window.Urls.get_node_diff(),
+                data:  JSON.stringify({'channel_id': self.id}),
+                success: function(data) {
+                    nodes = JSON.parse(data);
+                    resolve({
+                        "original" : new ContentNodeCollection(JSON.parse(nodes.original)),
+                        "changed" : new ContentNodeCollection(JSON.parse(nodes.changed))
+                    });
+                },
+                error:reject
+            });
+        });
+    },
+    sync_channel: function(options){
```

Comment:
sync_nodes takes in a list of node ids (more granular) whereas sync_channel takes in a channel id and syncs all of the nodes under it